intro-Linux/ch_uploading_files.tex

```diff
@@ -30,22 +39,19 @@ \section{Symlinks for data/scratch}
 lrwxrwxrwx 1 %\userid% %\userid% 34 Jun  5  2012 scratch -> %\scratchdir%
 \end{prompt}
 
-this will create 4 directories pointing to the respective storages
-
-\section{Editing: ``nano''}
+\section{Editing with \texttt{nano}}
 
 Nano is the simplest editor available on Linux. To open Nano, just type
-``nano''. To edit a file, you use ``nano the\_file\_to\_edit.txt''. You will
+\verb|nano|. To edit a file, you use \verb|nano the_file_to_edit.txt|. You will
 be presented with the contents of the file and a menu at the bottom with
-commands like ``\^{}O Write Out''. The ``\^{}'' is the Control key. So ``\^{}O'' means
-``Ctrl-O''. The main commands are:
+commands like \verb|^O Write Out| The \verb|^| is the Control key. So \verb|^O| means
+\verb|Ctrl-O|. The main commands are:
 
 \begin{enumerate}
-\item Open ("Read"): ``\^{}R''
-\item Save ("Write Out"): ``\^{}O''
-\item Exit: ``\^{}X''
+\item Open ("Read"): \verb|^R|
+\item Save ("Write Out"): \verb|^O|
+\item Exit: \verb|^X|
 \end{enumerate}
 
-More advanced editors (beyond the scope of this page) are ``vim'' and ``emacs''.
-
-
+More advanced editors (beyond the scope of this page) are \verb|vim| and \verb|emacs|.
+A simple tutorial on how to get started with \verb|vim| can be found \href{https://www.openvim.com/}{here}.
```

Comment:
I wouldn't hide the URL behind `here` in this case, but do this:

```
can be found at \url{https://www.openvim.com}.
```

Reply:
Done